CONTRIBUTING.md

```diff
@@ -102,7 +102,14 @@ If you've already signed a CLA but are still getting bothered by the awfully ins
 
 ## Set up
 
+You'll need at least [**Node** >= `18.15.0`](https://nodejs.org/en) installed on your machine.
+
+Once the minimum Node version is installed, you can continue with the rest of the repo setup.
+
 ```bash
+node -v
+v18.15.0
```

Comment:
In context with the rest of this script, this line could be interpreted as a command to type in.
Maybe make it a shell comment? e.g.:  `# v18.15.0`.

Reply:
Considering the context, I think it's fine. :)